csharp/Profiler/Trace.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+
+namespace Profiler
+{
+    /// <summary>
+    /// Trace-Script output type.
+    /// </summary>
+    public class Trace
+    {
+        public Profiler.LineProfile[] Top50Duration { get; set; }
```

Comment:
Idea: We might consider making this property doing the query lazily on demand and saving the result. Would save us some processing of the result, but user would have to "wait" for their result on the first query to the given property. But it still would be faster to do here than in powershell code even if it was not lazy.

Reply:
I like it. Users will probably stick to 1-2 views anyway, so saving some time is always nice. 

Thinking we can put it on the todo for the issue/PR that moves processing to c# at some point.